### DISCUSSION TITLE
Missing dependencies for gradle-completion when bash-completion@2 is installed

### DISCUSSION THREAD

**Discussion by Mahoney** [Homebrew/discussions#1017] [Casks]
Hi,

I have `gradle-completion` & `bash-completion@2` installed. When I run `brew doctor` I get warned about not having `bash-completion` installed, and `brew missing` outputs `gradle-completion: bash-completion`, as the formula shows:
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/gradle-completion.rb#L11

In practice it all works fine - I get tab completion when using `gradle`.

I presume the formula needs updating; can anyone give me a pointer as to how it should look? 

Thanks!

**Comment by SMillerDev** [comment]
Bash completion is not the same as bash completion 2. So this message is correct. Just like installing Java 15 won't get rid of a Java 8 requirement.

**Reply by Mahoney** [reply]
But `bash-completion` isn't actually required for `gradle-completion` to work - `bash-completion@2` seems to pull in the completion files fine. The other two completion casks I have installed (`maven-completion` and `docker-completion`) just seem not to specify a requirement at all; are they right, and `gradle-completion` wrong, or vice versa?
